modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-impl/src/main/java/com/liferay/headless/admin/taxonomy/internal/resource/v1_0/TaxonomyCategoryResourceImpl.java

```diff
@@ -354,7 +355,7 @@ public TaxonomyCategory postTaxonomyVocabularyTaxonomyCategory(
 			taxonomyVocabularyId);
 
 		return _addTaxonomyCategory(
-			assetVocabulary.getGroupId(),
+			null, assetVocabulary.getGroupId(),
```

Comment:
Same here. We should get the taxonomyCategory.getExternalReferenceCode() instead of null

Reply:
![shame on me](https://media.giphy.com/media/ws6SI3Hnrje5q/giphy.gif)

Reply:
I'm a bigger fan of the cone of shame. There is no shame if nobody can recognize you...

![Cone of shame](https://media3.giphy.com/media/AcsHgMcf26pva/giphy.gif?cid=ecf05e47chvbw840br4ulbvmzlzk526u8g2wxjy4q2vci783&rid=giphy.gif&ct=g)